### ISSUE TITLE
visual studio 15 won't load the extension anymore?

### ISSUE THREAD

**Issue by andmer** [Caphyon/clang-power-tools#464]
```
  <entry>
    <record>89</record>
    <time>2018/11/12 18:52:02.428</time>
    <type>Information</type>
    <source>VisualStudio</source>
    <description>Begin package load [RunClangPowerToolsPackage]</description>
    <guid>{F564F9D3-01AE-493E-883B-18DEEBDB975E}</guid>
  </entry>
  <entry>
    <record>90</record>
    <time>2018/11/12 18:52:02.447</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>SetSite failed for package [RunClangPowerToolsPackage]Source: &apos;ClangPowerTools&apos; Description: Could not load file or assembly &apos;Microsoft.VisualStudio.Threading, Version=15.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;System.IO.FileNotFoundException: Could not load file or assembly &apos;Microsoft.VisualStudio.Threading, Version=15.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;File name: &apos;Microsoft.VisualStudio.Threading, Version=15.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos;&#x000D;&#x000A;   at ClangPowerTools.RunClangPowerToolsPackage.InitializeAsync(CancellationToken cancellationToken, IProgress`1 progress)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.AsyncPackage.&lt;&gt;c__DisplayClass16_0.&lt;&lt;Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize&gt;b__1&gt;d.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.GetResult()&#x000D;&#x000A;&#x000D;&#x000A;WRN: Assembly binding logging is turned OFF.&#x000D;&#x000A;To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.&#x000D;&#x000A;Note: There is some performance penalty associated with assembly bind failure logging.&#x000D;&#x000A;To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].&#x000D;&#x000A;</description>
    <guid>{F564F9D3-01AE-493E-883B-18DEEBDB975E}</guid>
    <hr>80070002</hr>
    <errorinfo></errorinfo>
  </entry>
  <entry>
    <record>91</record>
    <time>2018/11/12 18:52:02.461</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [RunClangPowerToolsPackage]</description>
    <guid>{F564F9D3-01AE-493E-883B-18DEEBDB975E}</guid>
    <hr>80070002</hr>
    <errorinfo></errorinfo>
  </entry>



```

**Comment by dgagnon73** [comment]
Same here: it was loading fine when I installed it two weeks ago but not anymore. Must have updated automatically on me..

**Comment by Enache-Ionut** [comment]
Hello,

This issue was fixed and it is available in the new Clang Power Tools version ([v4.3](https://marketplace.visualstudio.com/items?itemName=caphyon.ClangPowerTools)) just released. 
Thank you for your feedback 😃
